lib/features/login/presentation/widgets/login_message_widget.dart

```diff
@@ -54,6 +55,10 @@ class LoginMessageWidget extends StatelessWidget {
               } else if (failure is GetTokenOIDCFailure && failure.exception is NoSuitableBrowserForOIDCException) {
                 return AppLocalizations.of(context).noSuitableBrowserForOIDC;
               } else if (failure is FeatureFailure) {
+                if (failure.exception is ConnectionError) {
```

Comment:
IMO, It should be exactly `NoNetworkError`

Reply:
Done